python/paddle/fleet/base/role_maker.py

```diff
@@ -286,6 +294,14 @@ def server_num(self):
             self.generate_role()
         return self._trainers_num
 
+    def node_num(self):
+        """
+        return the training node number
+        """
+        if not self._role_is_generated:
+            self.generate_role()
+        return self._node_num
```

Comment:
`node_num` define twice?

Reply:
The first node_num function is defined in `RoleMakerBase `, and the second node_num function is defined in `PaddleCloudRoleMaker `